src/Spa.lean

```diff
@@ -189,7 +189,7 @@ noncomputable def s_inv_aux (r1 r2 : rational_open_data A) (h : r1 ≤ r2) : uni
     refl,
 end)
 
--- Spa.rational_open_data.localization_map : the map between the uncompleted rings A(T1/s1)->A(T2/s2)
+-- spa.rational_open_data.localization_map : the map between the uncompleted rings A(T1/s1)->A(T2/s2)
```

Comment:
Isn't this comment duplicating the docstring?

Reply:
Removed the comment